src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesNodeCommandGroupHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    /// <summary>
+    /// Command handler that provides special handling for standard commands on dependency nodes.
+    /// </summary>
+    /// <remarks>
+    /// CPS provides a handler for the same commands, however it does not know anything special about dependency nodes.
+    /// </remarks>
+    [ExportCommandGroup(VSConstants.CMDSETID.StandardCommandSet2K_string)]
+    [AppliesTo(ProjectCapability.DependenciesTree)]
+    [Order(10)]
+    internal sealed class DependenciesNodeCommandGroupHandler : ICommandGroupHandler
```

Comment:
We have a concept called ProjectCommand and a derived type that simplifies this ceremony.

Reply:
`AbstractSingleNodeProjectCommand` with `ProjectCommandAttribute` looks like what you mean. That'd reduce the code a little, though makes it async. I've updated it.